src/sonic-frr/Makefile

```diff
@@ -4,17 +4,18 @@ SHELL = /bin/bash
 
 MAIN_TARGET = $(FRR)
 DERIVED_TARGET = $(FRR_PYTHONTOOLS) $(FRR_DBG) $(FRR_SNMP) $(FRR_SNMP_DBG)
+BRANCH = $(shell date +%Y%m%d\.%H%M%S)
```

Comment:
This will checkout a different branch each time FRR is built. Why is this necessary? The `stg undo` and `git clean -xfdf` should be enough to reset the repo, is it not?

Reply:
Otherwise I'll get 
`stg init: work: branch already initialized`

Reply:
Can you not just delete the branch during the cleanup?

Reply:
That doesn't help.
stg creates work.stgit branch and something else.